src/views/AuthoritiesSearch/AuthoritiesSearch.js

```diff
@@ -179,7 +206,11 @@ const AuthoritiesSearch = ({
     );
   };
 
-  const options = Object.values(searchResultListColumns).map((option) => ({
+  const sortableSearchResultListColumns = { ...searchResultListColumns };
```

Comment:
Let's just create a separate variable and list sortable columns like:
```
const sortableSearchResultListColumns = [
  searchResultListColumns.AUTH_REF_TYPE,
  searchResultListColumns.HEADING_REF,
  searchResultListColumns.HEADING_TYPE,
}
```
it's will be more declarative and easier to read than mutating an object